scripts/serve-dev.bash

```diff
@@ -0,0 +1,2 @@
+fast-live-reload --serve http://localhost:4200 --delay 1500 --add-path constrollers --add-path middleware --add-path src --add-path public --add-path views &
```

Comment:
This is not ideal because i hardcoded the port in here.  Ideally it should parse .env and use the port listed there.